IoTHubManager/HttpRequestWrapper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
```

Comment:
should this go into generic helper project?

Reply:
I m not sure if everyone needs this.

Reply:
It's generic enough that I would move it

Reply:
Ok